src/components/data-exploration/generic-gene-table/CreateCellSetModal.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {
+  Modal, Form, Button, Select, InputNumber, Row, Alert, Col,
+} from 'antd';
+import PropTypes from 'prop-types';
+
+const formItemStyle = { margin: '0.375rem 0' };
+
+const CreateCellSetModal = (props) => {
+  const { selectedGenes, onCancel } = props;
+  const [form] = Form.useForm();
+  const intialFormValues = selectedGenes.map((gene) => ({ selectedGenes: gene }));
+
+  const comparisonOptions = [{
+    value: 'greaterThan',
+    label: 'Greater than',
+  }, {
+    value: 'lessThan',
+    label: 'Less than',
+  },
+  ];
+
+  return (
+    <Modal
+      visible
+      title='Create a new cell set based on gene expression'
+      onCancel={onCancel}
+      footer={[<Button type='primary'>Create</Button>]}
+    >
+      <Form
+        form={form}
+        size='middle'
+        style={{ marginBottom: '1rem' }}
+      >
+        <Form.List
+          name='filterForm'
+          initialValue={intialFormValues}
+        >
+          {(fields) => (
+            <div style={{ maxHeight: '60vh', overflowY: 'auto', overflowX: 'hidden' }}>
+              {fields.map((field, index) => {
+                const { selectedGenes: formSelectedGenes } = form.getFieldValue('filterForm')[index];
+                return (
+                  <Row gutter={8}>
+                    <Col flex='auto'>
+                      <Form.Item
+                        name={[field.name, 'selectedGenes']}
+                        style={formItemStyle}
+                      >
+                        <strong>{formSelectedGenes}</strong>
+                      </Form.Item>
+                    </Col>
+                    <Col>
+                      <Form.Item
+                        name={[field.name, 'comparison']}
+                        style={formItemStyle}
+                      >
+                        <Select
+                          placeholder='Select comparison'
+                          defaultValue='greaterThan'
```

Comment:
instead of hardcoding `greaterThan`, you can choose the first element out of `comparisoOptions`